esmvaltool/utils/cmorizers/obs/cmorize_obs_Landschutzer2014.py

```diff
@@ -0,0 +1,117 @@
+"""
+# #############################################################################
+# ESMValTool CMORizer for WOA data
+# #############################################################################
+#
+# Tier
+#    Tier 2: other freely-available dataset.
+#
+# Source
+#    ftp://ftp.nodc.noaa.gov/nodc/archive/arc0105/0160558/1.1/data/0-data/
+#
+# Last access
+#    20190227
+#
+# Download and processing instructions
+#    Download the following file in ${RAWOBS}/Tier2/Landschutzer2014:
+#     pco2_1998-2011_ETH_SOM-FFN_CDIAC_G05.nc
+#      
+#
+# Modification history
+#    20190227-A_lova_to: written.
+#
+# #############################################################################
+"""
+
+import datetime
+import logging
+import os
+import glob
+
+import iris
+import netCDF4 as nc
+from cf_units import Unit
+import numpy as np
+
+from .utilities import (_add_metadata,
+                        _convert_timeunits,
+                        _fix_coords,
+                        _read_cmor_config,
+                        _roll_cube_data,
+                        _save_variable)
+
+logger = logging.getLogger(__name__)
+
+# read in CMOR configuration
+cfg = _read_cmor_config('Landschutzer2014.yml')
+proj = cfg['proj']
+timestamp = datetime.datetime.utcnow()
+timestamp_format = "%Y-%m-%d %H:%M:%S"
+now_time = timestamp.strftime(timestamp_format)
+proj['metadata_attributes']['CMORcreated'] = now_time
+VAR_TO_CMOR = cfg['VAR_TO_CMOR']
+VAR_TO_FILENAME = cfg['VAR_TO_FILENAME']
+STANDARD_NAMES = cfg['STANDARD_NAMES']
+LONG_NAMES = cfg['LONG_NAMES']
+NC4_ZIP = cfg['NC4_ZIP']
+# 
+ALLVARS=list(VAR_TO_CMOR.keys())
+
+
+def _fix_metadata(cube, var):
+    """Fix all aspects of metadata for different vars."""
+    logger.info("Fixing units...")
+    if var in ['fgco2',] :
+        cube.units = Unit('kg m-2 s-1')
+    if var in ['spco2','dpco2' ] :
+        cube.units = Unit('Pa')
+    return cube
+
+
+def _fix_data(cube, var):
+    """Specific data fixes for different variables."""
+    logger.info("Fixing data ...")
+    # fix for missing value definition 
+    cube.data=np.ma.masked_values(cube.data, cube.data.fill_value)
+    if var in ['fgco2'] :
+        """ Assume standard year 365_day"""
+        cube.data = cube.data * -12.01 / 1000. / (86400. * 365.)   # Reverse flux sign & Convert from mol m^{-2} yr^{-1} to kg m-2 s-1
+        dict.__setitem__(cube.attributes, 'positive', 'down')
```

Comment:
confused as to why you are setting positive: down as attribute to the cube itself - did you mean to do it for the verical coordinate? Also, if you want to set an attribute for a cube just do `cube.attributes['ATTR'] = 'VALUE'`

Reply:
This comes from the CMIP data request for the variable [fgco2](http://clipc-services.ceda.ac.uk/dreq/u/baa0d420-e5dd-11e5-8482-ac72891c3257.html) that is a flux form the air to the sea and thus it is necessary to report the direction of positive values of the flux as 'down'. Moreover, since 'positive' it is not a cube standard attribute I had to add also the option local_keys to add it to the saved variable.

I edited it as a dictionary, but I see that your correction is much better and I will change the code!